cluster/os-3.9.0/README.md

```diff
@@ -28,10 +24,32 @@ node01    Ready     master    1h        v1.9.1+a0ce1bc657
 node02    Ready     <none>    46s       v1.9.1+a0ce1bc657
 ```
 
+## OpenShift Web Console
+
+If you want to get access to OpenShift web console you will need to add line to `/etc/hosts`
+```bash
+echo "127.0.0.1 node01" >> /etc/hosts
+```
+
+and run your cluster like this:
+
+```bash
+export KUBEVIRT_PROVIDER=os-3.9.0
+export KUBEVIRT_NUM_NODES=2 # master + one nodes
+export KUBEVIRT_PROVIDER_EXTRA_ARGS="--ocp-port 8443"
```

Comment:
We could also opt-out for ci to avoid port clashes there. @davidvossel thoughts?

Reply:
do port clashes actually occur?  I'd rather keep this as simple as possible. if we don't have to introduce another var devs need to set to get functionality, that would be best.

Reply:
Fine for me. Then let's opt out in the CI script.

Reply:
Addressed. Let me know if you find something else.